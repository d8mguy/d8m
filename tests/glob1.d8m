// Run the Glob function in io/fs pkg, also using os.DirFS
import go "io/fs"
import go "os"
val foo = \imp() {
    val rslt = fs.Glob(os.DirFS("/Users/curtisabbott/go/src/github.com/curtisabbott/d8mingo/tests"), "bintree.*")
    if(rslt.err != nil) exit(rslt.err)
    rslt.ok
}
foo()
