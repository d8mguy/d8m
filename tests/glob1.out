[bintree.d8m, bintree.go, bintree.out]
